main.js

```diff
@@ -0,0 +1,34 @@
+
+var number = document.getElementById("number");
+
+var characterType = document.getElementById("characterType");
+var btn = document.getElementById("btn");
+btn.addEventListener("click", treeBuild);
+
+function treeBuild (event) {
+	if (number.value === "" || characterType.value === "") {
+		alert("You need to type something");
+	}
+for (var i = 1; i < number.value; i++) {
```

Comment:
The code looks good. The height is one line short when it runs though. It'll be a small change to line 12 to fix it.